sdk/monitor/azure-monitor-opentelemetry-exporter/tests/test_base_exporter.py

```diff
@@ -208,6 +209,12 @@ def test_transmission_200(self):
             result = self._base._transmit(self._envelopes_to_export)
         self.assertEqual(result, ExportResult.SUCCESS)
 
+    @mock.patch.dict(
+        os.environ,
+        {
+        "APPLICATIONINSIGHTS_STATSBEAT_DISABLED_ALL": "false",
+        }
+    )
```

Comment:
Is there a difference between mocking the envvar like this and assigning it like you do in other places?

Reply:
Mocking the envvar will only set the variable for that test. Assigning it like in `test_logs` will set the env var for the entire test suite.

Reply:
Got it